cypress/integration/fee-fine/fee-fine-history.spec.js

```diff
@@ -0,0 +1,139 @@
+import { HTML } from '../../../interactors';
+
+//The test fails because of the bug.
+describe('Fee/Fine history ', () => {
+  const userData = {
+    permissions: [
+      permissions.uiUsersView.gui,
+      permissions.uiUsersfeefinesCRUD.gui,
+      permissions.uiUsersViewServicePoints.gui,
+      permissions.uiUsersfeefinesView.gui,
+      permissions.uiUsersManualCharge.gui,
+      permissions.uiUsersManualPay.gui,
+      permissions.uiUserAccounts.gui
+    ],
+  };
+  const ownerData = {};
+  let feeFineType = {};
+  const paymentMethod = {};
+  const waiveReason = {};
+  const refundReason = {};
+  const transferAccount = getNewTransferAccount();
+  let feeFineAccount;
+  let servicePointId;
+  const newStaffInfoMessage = 'information to check';
+
+  before('Create owner, transfer account, feeFineType, paymentMethod, waiveReason, refundReason, user, feeFine', () => {
+    cy.getAdminToken();
+    ServicePoints.getViaApi({ limit: 1 }).then((servicePoints) => { servicePointId = servicePoints[0].id; });
+
+    UsersOwners.createViaApi(getNewOwner('owner')).then(({ id, ownerName }) => {
+      ownerData.name = ownerName;
+      ownerData.id = id;
+    })
+      .then(() => {
+        TransferAccounts.createViaApi({ ...transferAccount, ownerId: ownerData.id }); // is not created
+        ManualCharges.createViaApi({ ...ManualCharges.defaultFeeFineType, ownerId: ownerData.id }).then(manualCharge => {
+          feeFineType = { id:  manualCharge.id, name: manualCharge.feeFineType, amount: manualCharge.amount };
+        });
+        PaymentMethods.createViaApi(ownerData.id).then(({ name, id }) => {
+          paymentMethod.name = name;
+          paymentMethod.id = id;
+        });
+        WaiveReasons.createViaApi(getNewWaiveReason()).then(res => {
+          waiveReason.id = res.body.id;
+        });
+
+        RefundReasons.createViaApi(getNewRefundReason()).then(res => {
+          console.log(JSON.stringify(res));
+          refundReason.id = res.body.id;
+        });
+
+        cy.createTempUser(userData.permissions)
+          .then(userProperties => {
+            userData.username = userProperties.username;
+            userData.password = userProperties.password;
+            userData.userId = userProperties.userId;
+            userData.barcode = userProperties.barcode;
+            userData.firstName = userProperties.firstName;
+          })
+          .then(() => {
+            UserEdit.addServicePointViaApi(servicePointId, userData.userId);
+            feeFineAccount = { ownerId: ownerData.id,
+              feeFineId: feeFineType.id,
+              amount: feeFineType.amount,
+              userId: userData.userId,
+              feeFineType: feeFineType.name,
+              createdAt: servicePointId,
+              dateAction: moment.utc().format(),
+              feeFineOwner: ownerData.name,
+              source: 'ADMINISTRATOR, DIKU' };
+            NewFeeFine.createViaApi(feeFineAccount).then((feeFineAccountId) => {
```

Comment:
can't see clearing of feeFineAccount. please recheck this

Reply:
I have added the remove method